### PR TITLE
Feat: Style Links

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
Styles our three types of links:
- Link (default)
- Link Inline
- Link Display
- Link Footer
- Link Inverse (to use on dark backgrounds)

## How it works? 
|Type|Image|
|-|-|
|Default|image|
|Inline|image|
|Display|image|
|Footer|image|
|Inverse|image|

## How to test it?
You can see it working on the `/pattern-library` page

### CODE REVIEW

**Review by ArthurTriis1 — APPROVED** [review]

File: src/components/ui/Link/link.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-link] {
```
Comment (hellofanny) [inline src/components/ui/Link/link.scss:3]: Just to confirm, we are applying this style for all the links, right?

File: src/components/ui/Link/link.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+// Links ////////////////////////////
+
+.link {
+  color: var(--color-link);
+  line-height: 1.5;
+}
+
+.link-display {
+  color: var(--color-secondary-4);
```
Comment (hellofanny) [inline src/components/ui/Link/link.scss:20]: Does make sense to create a semantic variable for this color?

File: src/components/ui/Link/link.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  line-height: 1.5;
+}
+
+.link-inline {
```
Comment (hellofanny) [inline src/components/ui/Link/link.scss:24]: hmm, here I would suggest or follow BEM pattern to name these classes or create `data-atribule` with variant to style it. What do you think?

**Review by hellofanny — CHANGES_REQUESTED** [review]
Nicee @renatamottam! I've left just a comment 👍  Lemme know what you think and If I can help with that!

**Review by israelswf — APPROVED** [review]
Looking and working nicely to me ✨

File: src/components/ui/Link/link.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  color: var(--color-link-inverse);
+  &:visited { color: var(--color-link-inverse); }
+}
+
+.link {
+  color: var(--color-link);
+  line-height: 1.5;
+}
+
+.link-display {
```
Comment (hellofanny) [inline src/components/ui/Link/link.scss:24]: Hmm.. if we are going to follow BEM syntax, are we considering these following classes elements or modifies? 
`link__display` or `link--display`?

Comment (hellofanny) [reply src/components/ui/Link/link.scss:24]: Or maybe we want to pass all this classes as variants?

**Review by hellofanny — CHANGES_REQUESTED** [review]
hey, I left a comment regarding the variants naming, can you take a look please?

**Review by lucasfp13 — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]

**Pull request merged**